src/components/CodeMirror.ts

```diff
@@ -433,8 +433,10 @@ export default defineComponent({
     watch(
       () => props.modelValue,
       async value => {
-        if (view.value.composing) {
-          // IME fix
+        if (
+          view.value.composing || // IME fix
+          view.value.state.doc.toJSON().join('\n') === value // don't need to update
```

Comment:
This can be simplified as `view.value.state.doc.toString() === value` according to https://codemirror.net/docs/ref/#state.Text.toString.

Reply:
Note, 1. `toString` is only added to index.d.ts 6.2.x though it works fine on 6.1.4; 2. regardless the version, ESLint will (wrongly?) report error no-base-to-string.

Reply:
It works well but I got a type error, seems there are some type definition issues

![image](https://github.com/logue/vue-codemirror6/assets/32098252/3da575c8-ad99-45ff-b951-84185d870113)

Reply:
yeah, I noticed the same issue. Actually, I commented after I found the issue but forgot the commit the comment :)

Reply:
Then should we still use it like "(view.value.state.doc as any).toString()"? I would prefer the current one, we can fix it after bump the version.